src/ElectricPokemon.java

```diff
@@ -0,0 +1,32 @@
+import java.util.ArrayList;
+
+public class ElectricPokemon extends Pokemon {
+
+    public static ArrayList<ElectricPokemon> electricPokemons = new ArrayList<>();
+
+    public ElectricPokemon(String _name, int _level ){
```

Comment:
De _name is een python notatie en heeft daar ook een nut. In java heeft die underscore geen nut (zover ik weet en ik kan er ook niks over vinden)